exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/join/HashJoinBatch.java

```diff
@@ -1250,8 +1263,10 @@ public HashJoinBatch(HashJoinPOP popConfig, FragmentContext context,
     partitions = new HashPartition[0];
 
     // get the output batch size from config.
-    final int configuredBatchSize = (int) context.getOptions().getOption(ExecConstants.OUTPUT_BATCH_SIZE_VALIDATOR);
-    final double avail_mem_factor = (double) context.getOptions().getOption(ExecConstants.OUTPUT_BATCH_SIZE_AVAIL_MEM_FACTOR_VALIDATOR);
+    final int configuredBatchSize = (int) context.getOptions().getOption(
```

Comment:
If option is long why we use int?

Reply:
This is original code, I just split the line for legibility. One could argue that a patch size larger than 2 GB would be excessive, I suppose.